upgrade/helpers/tasks.py

```diff
@@ -179,7 +193,16 @@ def _sync_capsule_subscription_to_capsule_ak(ak):
             name=rhelcontents['capsule']['repofull'].format(
                 cap_ver=to_version, os_ver=os_ver, arch='x86_64')
         ).search(query={'organization_id': org.id, 'per_page': 100})[0]
-    call_entity_method_with_timeout(entities.Repository(id=cap_repo.id).sync, timeout=2500)
+        logger.info("Capsule Repository's repofull {} search completed successfully".
+                    format(rhelcontents['capsule']['repofull']
+                           .format(cap_ver=to_version, os_ver=os_ver, arch='x86_64')))
+    logger.info("Entities repository sync operation started successfully for name {}".
+                format(cap_repo.name))
+    start_time = job_execution_time("Entity repository sync")
+    call_entity_method_with_timeout(entities.Repository(id=cap_repo.id).sync, timeout=4000)
```

Comment:
same as above ^^

Reply:
To avoid further any timeout-related failure, we analyzed the log after sync.